Code/Framework/AzFramework/AzFramework/TargetManagement/NeighborhoodAPI.h

```diff
@@ -46,59 +46,14 @@ namespace Neighborhood {
         virtual ~NeighborhoodEvents() {}
 
         // used to advertise a node and its capabilities
```

Comment:
Can we convert comments to autogen comments please esp if you've updated the methods?